cmake/Platform/Common/Install_common.cmake

```diff
@@ -407,6 +408,7 @@ endif()
         "${CREATE_ALIASES_PLACEHOLDER}"
         "${GEM_VARIANT_TO_LOAD_PLACEHOLDER}"
         "${ENABLE_GEMS_PLACEHOLDER}"
+        "${O3DE_INSTALL_CODE_PLACEHOLDER}"
```

Comment:
We may want to inject the subdirectory install code before the CMake script which adds the `ly_add_target` commands.
"${all_configured_targets}"`

Reply:
Done.